### PR TITLE
Clean the submit-handler and router-handler modules

### PR DESCRIPTION
**Pull request by barakman**

### CODE REVIEW

File: fastlane_bot/tests/test_043_TestEmptyCarbonOrders.py [hunk 1]
```diff
@@ -13,17 +13,13 @@
...
 from fastlane_bot.events.exchanges import UniswapV2, UniswapV3,  CarbonV1, BancorV3
 from fastlane_bot.events.interface import QueryInterface
-from fastlane_bot.helpers.poolandtokens import PoolAndTokens
-from fastlane_bot.helpers import TradeInstruction, TxReceiptHandler, TxRouteHandler, TxSubmitHandler, TxHelpers, TxHelper
 from fastlane_bot.events.managers.manager import Manager
 from fastlane_bot.events.interface import QueryInterface
 from joblib import Parallel, delayed
-from dataclasses import dataclass, asdict, field
-import pytest
+from dataclasses import asdict
```
Comment (sklbancor) [inline fastlane_bot/tests/test_043_TestEmptyCarbonOrders.py:22]: looks like you are not using asdict?

**Review by sklbancor — DISMISSED** [review]

**Comment by sklbancor** [comment]
seems uncontroversial

File: fastlane_bot/bot.py [hunk 2]
```diff
@@ -133,9 +125,6 @@ def __post_init__(self):
 
         if self.TxRouteHandlerClass is None:
             self.TxRouteHandlerClass = TxRouteHandler
-        assert issubclass(
```
Comment (sklbancor) [inline fastlane_bot/bot.py:136]: why do you remove this assertion?

**Review by sklbancor — APPROVED** [review]

File: fastlane_bot/bot.py (hunk 2 above)
Comment (barakman) [reply fastlane_bot/bot.py:136]: Because I've removed the (empty) class `TxRouteHandlerBase`

**Pull request merged**